nixos/tests/nebula/reload.nix

```
@@ -72,7 +72,7 @@ in
 
       # Restart nebula to pick up the keys.
       lighthouse.systemctl("restart nebula@smoke.service")
-      lighthouse.succeed("ping -c5 10.0.100.1")
+      lighthouse.wait_until_succeeds("ping -c1 -W1 10.0.100.1")
```

```suggestion
      lighthouse.wait_until_succeeds("ping -c1 -W1 10.0.100.1", timeout=10)
```
Probably an oversight? I don't think we need the default 900s timeout here if it was previously just 5 pings.

---

Yup, missed the other file on my find/replace. Fixed.